lib/react/rails/controller_renderer.rb

```diff
@@ -0,0 +1,17 @@
+class React::Rails::ControllerRenderer
+  include React::Rails::ViewHelper
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::TextHelper
+
+  attr_accessor :output_buffer
```

Comment:
cool, I was trying to refactor the view helper a while back and got scared off by how things kept _not working_, maybe i'll give it a try